types/react-native-zeroconf/index.d.ts

```diff
@@ -29,25 +29,41 @@ export interface Service {
     };
 }
 
+export type ImplType = "NSD" | "DNSSD";
```

Comment:
Looks like Zeroconf also [exports](https://github.com/balthazar/react-native-zeroconf/blob/master/src/index.js#L6) `ImplType` as a dict, which might be nice to add to the typings as well.

```suggestion
export declare const ImplType: {
  readonly NSD: 'NSD',
  readonly DNSSD: 'DNSSD',
};
export type ImplType = (typeof ImplType)[keyof typeof ImplType];
```